components/header_link.js

```diff
@@ -64,6 +65,7 @@ class HeaderLink extends Component {
               ? cx(style, grey, className)
               : cx(style, className)
           }
+          target={target}
```

Comment:
I think it would be better to leave `target` out of this file - the prop will be passed through automatically with `{...otherProps}`. Try reverting your changes to this component - the behaviour should be the same.

Reply:
I tried removing line 68 along with other lines I had added, but this causes it to open in the same tab, not a new one.

Reply:
Okay it should be working now - the ... on line 54 was missing